source/SpinalHDL/Data types/bool.rst

```diff
@@ -33,14 +33,16 @@ The syntax to declare a boolean value is as follows: (everything between [] is o
      - Create a Bool assigned with ``false``
      - Bool
    * - Bool(value: Boolean)
-     - Create a Bool assigned with a Scala Boolean(true, false)
+     - Create a Bool assigned with a value from a Scala Boolean type (true,
+       false).  This implicitly converts to ``True`` or ``False`` as
+       necessary during elboration phase of HDL generation.
      - Bool
 
 
 .. code-block:: scala
 
-   val myBool_1 = Bool()          // Create a Bool
-   myBool_1 := False            // := is the assignment operator
+   val myBool_1 = Bool()        // Create a Bool
+   myBool_1 := False            // := is the assignment operator (like verilog <=)
```

Comment:
If we want to add that information we should make a "Help for Verilog people" chapter rather than this one lone mention?

Reply:
I understand what you mean. But is this in the end a helpful equivalence? `:=` is used in places were verilog would use `<=` or `=`? But if you think it helps that much we can also keep it.

And no, I dont think we need "equal representation" ;-) VHDL equivalents have their own page where we have to space to actually mention the details.

Reply:
As someone still in their first year of using SpinalHDL, I would have found it useful to get a frame of reference.

With regards to `<=` and `=` differences, it does not matter for the purpose of frame of refrence, the most often used type in verilog assignment type is `<=` by maybe 99:1 in the verilog I have seen.  YMMV

Also the snippet in this code example box is really targeting a novice/beginner to intermediate user of SpinalHDL.  I think an intermediate user and above will glance at that code example box and move on, on the basis there is nothing to learn in there.